_includes/footer.html

```diff
@@ -12,8 +12,8 @@
     </a>
   </div>
   <div class="fx-col-100-xs fx-col-33-s fx-col-center-xs">
-    <a href="https://sourceforge.net/projects/llcon" target="_blank" rel="noreferrer">
-      <img src="https://sourceforge.net/sflogo.php?type=17&amp;group_id=158367" alt="{{ site.data.general.footer.alt.sourceforge }}" class="fx-center" id="sflogo" />
+    <a href="https://github.com/jamulussoftware/jamulus/discussions" target="_blank" rel="noreferrer">
+      <img src="https://icongr.am/fontawesome/question-circle-o.svg?size=35&color=currentColor" alt="{{ site.data.general.footer.alt.sourceforge }}" class="fx-center" id="sflogo" />
```

Comment:
Note: I think we’ll need to edit the alt text too. (See the _data) folder. I can do this too, if you want.